File: dotnet/react.mvc-core.postgres.dapper/TuduManager/src/TuduManayer.Domain/Todo/Create/Models/Todo.cs
using System;

namespace TuduManayer.Domain.Todo.Create.Models
{
    public class Todo
    {
        public string Title { get; }
        public string Description { get; }
        public DateTime CreationDate { get; }
        
        public Todo(string title, string description)
        {
            Title = title;
            Description = description;
            CreationDate = DateTime.Now;
        }
    }
}

File: java/TuduManager/domain/src/main/java/net/seymourpoler/tudumanager/domain/todo/create/models/Todo.java
package net.seymourpoler.tudumanager.domain.todo.create.models;

import java.time.LocalDateTime;

public class Todo {
    public final String title;
    public final String description;
    public final LocalDateTime creationDate;

    public Todo(String title, String description) {
        this.title = title;
        this.description = description;
        creationDate = LocalDateTime.now();
    }
}
